Hymns/Home/HomeView.swift

```diff
@@ -25,11 +25,23 @@ struct HomeView: View {
                 Text($0).font(.caption).padding(.top).padding(.leading)
             }
 
-            List(self.viewModel.songResults) { songResult in
-                NavigationLink(destination: songResult.destinationView) {
-                    SongResultView(viewModel: songResult)
+            if self.viewModel.isLoading {
+                VStack {
+                    Spacer()
+                    HStack {
+                        Spacer()
+                        ActivityIndicator().eraseToAnyView()
+                        Spacer()
+                    }
+                    Spacer()
```

Comment:
If you have a better way to center the activity indicator, let me know

Reply:
That should be fine.